content/Custom/Hubs/Web-Editor-Support.md

```diff
@@ -60,16 +60,19 @@ id: 9393241841308
     <a class="link-chevron-right" href="https://support.arduino.cc/hc/en-us/articles/4402764401554-Compilation-errors-when-uploading">If your sketch doesn’t compile</a>
   </div>
   <div class="info-item">
-    <img src="https://content.arduino.cc/assets/hc-stop.svg">
-    <p class="info-title">Can't access the Web Editor?</p>
+    <img src="https://content.arduino.cc/assets/hc-USB.svg">
+    <p class="info-title">Problems using the editor</p>
     <p>
-      Learn more about possible solutions for issues with accessing the Web
-      Editor:
+      As a first troubleshooting step when the Web Editor is not detecting
+      a board always make sure that the Arduino Create Agent is installed and
+      running. Learn more about:
     </p>
-    <a class="link-up-right" href="https://support.arduino.cc/hc/en-us/articles/8825427680796-If-you-can-t-access-the-Web-Editor">Read the guide</a>
+    <a class="link-chevron-right" href="https://support.arduino.cc/hc/en-us/articles/360018131160-If-your-board-is-not-detected-by-Arduino-Web-Editor">If the board is not recognized</a>
+    <a class="link-chevron-right" href="https://support.arduino.cc/hc/en-us/articles/360016466600-If-the-Arduino-Create-Agent-isn-t-detected">If the Arduino Create Agent isn't detected</a>
+    <a class="link-chevron-right" href="https://support.arduino.cc/hc/en-us/articles/8825427680796-If-you-can-t-access-the-Web-Editor">If you can't access the Web Editor</a>
   </div>
 </div>
-<h2 class="center hub">Resources</h2>
+<h2 id="h_01HD21Q0JXNFAQB70C4BB6DTAZ" class="center hub">Resources</h2>
 <div class="info-wrapper">
   <div class="info-item">
```

Comment:
This info-item should be deleted since it has the same information as the "Importing files" one.